selfdrive/controls/lib/longitudinal_mpc_lib/SConscript

```diff
@@ -66,7 +66,7 @@ lenv.Clean(generated_files, Dir(gen))
 generated_long = lenv.Command(generated_files,
                               source_list,
                               f"cd {Dir('.').abspath} && python3 long_mpc.py")
-lenv.Depends(generated_long, [msgq_python, common_python, opendbc_python])
+lenv.Depends(generated_long, [msgq_python, common_python, opendbc_python, pandad_python])
```

Comment:
Why is this needed?

Reply:
Forgot why. Maybe because I added pandad_python to all modules that depend on opendbc_python. It should not be needed here.

Reply:
I think it may be because radard uses it, and the MPC imports something from radard.

Reply:
update: remove them will cause compile errors:

  File "/home/dean/Projects/openpilot/selfdrive/controls/lib/longitudinal_mpc_lib/long_mpc.py", line 11, in <module>
    from openpilot.selfdrive.car.interfaces import ACCEL_MIN
  File "/home/dean/Projects/openpilot/openpilot/selfdrive/car/interfaces.py", line 21, in <module>
    from openpilot.selfdrive.pandad import can_capnp_to_list
  File "/home/dean/Projects/openpilot/openpilot/selfdrive/pandad/__init__.py", line 2, in <module>
    from openpilot.selfdrive.pandad.pandad_api_impl import can_list_to_can_capnp, can_capnp_to_list
ModuleNotFoundError: No module named 'openpilot.selfdrive.pandad.pandad_api_impl'

Reply:
I tried this, but realized long_mpc also imports from interfaces, so your approach is the simplest

```
class RadarInterfaceBase(ABC):
  def __init__(self, CP):
    self.rcp = None
    self.pts = {}
    self.delay = 0
    self.radar_ts = CP.radarTimeStep
    self.frame = 0

  def update(self, can_strings: list[bytes]):
    return self._update(can_capnp_to_list(can_strings))

  @abstractmethod
  def _update(self, can_list):
    pass

  def no_radar_response(self):
    self.frame += 1
    if (self.frame % int(100 * self.radar_ts)) == 0:
      return car.RadarData.new_message()
    return None
```